### ISSUE TITLE
An error at the end of a run

### ISSUE THREAD

**Issue by CTIfitness** [Anjok07/ultimatevocalremovergui#449]
Last Error Received:

Process: VR Architecture

If this error persists, please contact the developers with the error details.

Raw Error Details:

UnpicklingError: "invalid load key, '\x00'."
Traceback Error: "
  File "UVR.py", line 4592, in process_start
  File "separate.py", line 632, in seperate
  File "separate.py", line 832, in spec_to_wav
  File "lib_v5\spec_utils.py", line 347, in cmb_spectrogram_to_wave
  File "lib_v5\spec_utils.py", line 286, in spectrogram_to_wave
  File "librosa\util\decorators.py", line 88, in inner_f
  File "librosa\core\spectrum.py", line 416, in istft
  File "numba\core\dispatcher.py", line 439, in _compile_for_args
  File "numba\core\dispatcher.py", line 372, in _compile_for_args
  File "numba\core\dispatcher.py", line 891, in compile
  File "numba\core\caching.py", line 644, in load_overload
  File "numba\core\caching.py", line 651, in _load_overload
  File "numba\core\caching.py", line 495, in load
  File "numba\core\caching.py", line 512, in _load_index
"

Error Time Stamp [2023-03-14 22:05:41]

Full Application Settings:

vr_model: 1_HP-UVR
aggression_setting: 10
window_size: 320
batch_size: 4
crop_size: 256
is_tta: False
is_output_image: False
is_post_process: False
is_high_end_process: False
post_process_threshold: 0.2
vr_voc_inst_secondary_model: No Model Selected
vr_other_secondary_model: No Model Selected
vr_bass_secondary_model: No Model Selected
vr_drums_secondary_model: No Model Selected
vr_is_secondary_model_activate: False
vr_voc_inst_secondary_model_scale: 0.9
vr_other_secondary_model_scale: 0.7
vr_bass_secondary_model_scale: 0.5
vr_drums_secondary_model_scale: 0.5
demucs_model: Choose Model
segment: Default
overlap: 0.25
shifts: 2
chunks_demucs: Auto
margin_demucs: 44100
is_chunk_demucs: False
is_primary_stem_only_Demucs: False
is_secondary_stem_only_Demucs: False
is_split_mode: True
is_demucs_combine_stems: True
demucs_voc_inst_secondary_model: No Model Selected
demucs_other_secondary_model: No Model Selected
demucs_bass_secondary_model: No Model Selected
demucs_drums_secondary_model: No Model Selected
demucs_is_secondary_model_activate: False
demucs_voc_inst_secondary_model_scale: 0.9
demucs_other_secondary_model_scale: 0.7
demucs_bass_secondary_model_scale: 0.5
demucs_drums_secondary_model_scale: 0.5
demucs_pre_proc_model: No Model Selected
is_demucs_pre_proc_model_activate: False
is_demucs_pre_proc_model_inst_mix: False
mdx_net_model: Choose Model
chunks: Auto
margin: 44100
compensate: Auto
is_denoise: False
is_invert_spec: False
mdx_voc_inst_secondary_model: No Model Selected
mdx_other_secondary_model: No Model Selected
mdx_bass_secondary_model: No Model Selected
mdx_drums_secondary_model: No Model Selected
mdx_is_secondary_model_activate: False
mdx_voc_inst_secondary_model_scale: 0.9
mdx_other_secondary_model_scale: 0.7
mdx_bass_secondary_model_scale: 0.5
mdx_drums_secondary_model_scale: 0.5
is_save_all_outputs_ensemble: True
is_append_ensemble_name: False
chosen_audio_tool: Manual Ensemble
choose_algorithm: Min Spec
time_stretch_rate: 2.0
pitch_rate: 2.0
is_gpu_conversion: True
is_primary_stem_only: True
is_secondary_stem_only: False
is_testing_audio: False
is_add_model_name: False
is_accept_any_input: False
is_task_complete: False
is_normalization: False
is_create_model_folder: False
mp3_bit_set: 320k
save_format: FLAC
wav_type_set: PCM_16
help_hints_var: False
model_sample_mode: False
model_sample_mode_duration: 30
demucs_stems: All Stems

**Comment by sabaasa** [comment]
does this happen with every audio file you try? or is this a problem with one specific one

**Comment by CTIfitness** [comment]
Yes,whether it is CPU processing or GPU processing,this error will occur.All audio is like this,without exception.The previous version of 5.4 ran without problems. I want to try the 5.4 version, but I can't find the download link.

**Comment by CTIfitness** [comment]
> does this happen with every audio file you try? or is this a problem with one specific one

**Comment by CTIfitness** [comment]
> does this happen with every audio file you try? or is this a problem with one specific one

Yes,whether it is CPU processing or GPU processing,this error will occur.All audio is like this,without exception.The previous version of 5.4 ran without problems. I want to try the 5.4 version, but I can't find the download link.

**Comment by Anjok07** [comment]
Have you tried reinstalling UVR?

**Comment by CTIfitness** [comment]
I installed it on the D drive, but encountered continuous problems. I remember that there were no issues when installing it on the D drive before. Now that I have installed it on the C drive, there are no problems. I am wondering when it will support installation on non-system drives.
----- 原始邮件 -----
发件人：Anjok07 ***@***.***>
收件人：Anjok07/ultimatevocalremovergui ***@***.***>
抄送人：CTIfitness ***@***.***>,  Author ***@***.***>
主题：Re: [Anjok07/ultimatevocalremovergui] An error at the end of a run (Issue #449)
日期：2023年03月31日 19点35分


Have you tried reinstalling UVR?

&mdash;
